src/chartconfiguration/stackedchartconfiguration.ts

```diff
@@ -0,0 +1,107 @@
+export const PERCENTILEGROUPS = [
```

Comment:
I'd use camel case (standard for js/ts) or at least add underscore between words.